src/rendering/dashboardtextitem.cpp

```diff
@@ -85,4 +87,10 @@ DashboardTextItem::DashboardTextItem(const ghoul::Dictionary& dictionary, float
     _font = global::fontManager->font(_fontName, _fontSize);
 }
 
+void DashboardTextItem::render(glm::vec2& penPosition) {
```

Comment:
Curious, since I don't really understand it.  What is the "penPosition" ? While we're refactoring, could we maybe give this variable a more understandable name or add a Doxygen comment over the base render function to explain it?

Reply:
I kept any refactoring out of this PR on purpose or it would have grown into something much bigger.  Added a documentation entry to the root DashboardItem class

```
    /**
     * Renders this DashboardItem at the provided \p penPosition. The position indicates
     * where this DashboardItem should render itself and is provided in pixel-coordinates
     * with the top-left corner of the screen being at (0,0). Each derived subclass shall
     * update the \p penPosition according to the items that it should render. If, for
     * example, a single line of text is rendered, the \p penPosition shall be updated by
     * one line's worth of vertical separation.
     *
     * \p penPosition The position at which this DashboardItem should be rendered
     */
```